internal/service/enrollmentrequest.go

```diff
@@ -28,7 +27,7 @@ func validateAndCompleteEnrollmentRequest(enrollmentRequest *api.EnrollmentReque
 	return nil
 }
 
-func approveAndSignEnrollmentRequest(ca *crypto.CA, enrollmentRequest *api.EnrollmentRequest, approval *v1alpha1.EnrollmentRequestApproval) error {
+func approveAndSignEnrollmentRequest(ca *crypto.CA, enrollmentRequest *api.EnrollmentRequest, approval *api.EnrollmentRequestApproval) error {
```

Comment:
I believe we standardized on `v1alpha1` please drop this and if you see `api` usage update. The reason is it is more explicit in the code and updating versions is also more explicit in the diff.  :pray:

Reply:
makes sense. Updated